### PR TITLE
Two smart chessboards

### PR DESCRIPTION
**Pull request by WilmervH**
## Describe your project in 1-3 sentences

We are building two smart chessboards with online play and local play against an AI. The project will be done together with @fellu4485. 

## Hack Club Slack handle

Wilmy

## Checklist before submitting proposal:

- [X] I am a current high school student (or younger).
- [X] I have submitted an RSVP at <https://hackclub.com/winter>
- [X] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [X] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by fellu4485** [comment]
I will also participate in this project. My Hack Club Slack handle is: fellu

File: WilmervH.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+| 2x Raspberry pi              | Dependant on availability                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | up to 200 euros |
```
Comment (exu3) [inline WilmervH.md:49]: In order for Hack Club to give you money to purchase these, you'll need to provide a link to where you might find them. https://rpilocator.com/ may be helpful if you can find your country.

File: WilmervH.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+| 2x SD-card                   | [Muistikauppa.fi](https://www.muistikauppa.fi/tuote/kingston-canvas-select-plus-64gb-microsdxc-muistikortti/)                                                                                                                                                                                                                                                                                                                                                                                                                                                              | 12,80 €         |
```
Comment (exu3) [inline WilmervH.md:56]: Would you happen to have an SD card reader for these?

**Review by exu3 — CHANGES_REQUESTED** [review]
This is super neat! Your plan seems clear and sounds like a feasible project. I left a couple comments on your budget but everything else looks great.

**Comment by WilmervH** [comment]
Hi! I own a SD-card reader and have added it to the tools section. We will probably buy the Raspberry Pis from buyzero.de. They do not show shipping fees until checkout but I am sure the fees can not be higher than the remaining budget.

**Comment by exu3** [comment]
> Hi! I own a SD-card reader and have added it to the tools section. We will probably buy the Raspberry Pis from buyzero.de. They do not show shipping fees until checkout but I am sure the fees can not be higher than the remaining budget.

https://buyzero.de/en/products/raspberry-pi-4-model-b-8gb?variant=40326644498612 appears to be sold out. Do you think you could find one in stock and link it in your document?

Other than that, I think this project is good to move forward!

**Review by exu3 — APPROVED** [review]

**Review by MelSmith104 — COMMENTED** [review]
hey @WilmervH and @fellu4485 ! i love your project idea so much! i love playing chess and this seems like its a great blend of both programming and chess :) i can't wait to learn more about it! i'm dm'ing the two of you on slack right now so that we can chat about next steps!

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**